elements/pfe-codeblock/src/pfe-codeblock-material-dark.scss

```
@@ -0,0 +1,223 @@
+@import "../../pfe-sass/pfe-sass";
+
+//Define custom colors
+$custom-red: #FF6666;
+$custom-orange: #FE9A8C; //(red, #eee, etc.)
+$custom-yellow: #FFCB6B; //(height, width, cx, cy, etc.)
+$custom-green: #A5E844;
+$custom-teal: #80CBC4;
+$custom-blue: #73BCF7;
+$custom-purple: #C792EA;
+$custom-dark-gray: #B8BBBE; // (numbers and divider)
+$custom-gray: #B8BBBE; // (text)
+$custom-white: #FFF;
+$custom-black: #2f2f2f;
+$custom-hilight-bg: #363636;
+
+:host([pfe-c-codeblock-theme="dark"]) {
+	code[class*="language-"],
+	pre[class*="language-"] {
+		text-align: left;
+		white-space: pre;
+		word-spacing: normal;
+		word-break: normal;
+		word-wrap: normal;
+		color: $custom-gray;
+		background: $custom-black;
+		font-family: unquote(map-get($pfe-vars, font-family--code));
+		font-size: 1em;
+		line-height: 1.5em;
+
+		-moz-tab-size: 4;
+		-o-tab-size: 4;
+		tab-size: 4;
+
+		-webkit-hyphens: none;
+		-moz-hyphens: none;
+		-ms-hyphens: none;
+		hyphens: none;
+	}
+
+	code[class*="language-"]::-moz-selection,
+	pre[class*="language-"]::-moz-selection,
+	code[class*="language-"] ::-moz-selection,
+	pre[class*="language-"] ::-moz-selection {
+		background: $custom-hilight-bg;
+	}
+
+	code[class*="language-"]::selection,
+	pre[class*="language-"]::selection,
+	code[class*="language-"] ::selection,
+	pre[class*="language-"] ::selection {
+		background: $custom-hilight-bg;
+	}
+
+	:not(pre) > code[class*="language-"] {
+		white-space: normal;
+		border-radius: 0.2em;
+		padding: 0.1em;
+	}
+
+	pre[class*="language-"] {
+		overflow: auto;
+		position: relative;
+		margin: 0.5em 0;
+		padding: 1.25em 1em;
+	}
+
+	.language-css > code,
+	.language-sass > code,
+	.language-scss > code {
+		color: $custom-orange;
+	}
+
+	[class*="language-"] .namespace {
+		opacity: 0.7;
+	}
+
+	.token.atrule {
```

```suggestion
	.token.atrule {
```

Can you nest these styles under a `.token` heading like:
```
.token {
    &.atrule {
        color: $custom-purple;
    }
    &.attr-name {
        color: $custom-orange;
    }
}
```

---

I have made the suggested change